I.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,17 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
 // be sure to exclude the strings
+function getAverage(array) {
+    let numArray = [];
+    array.forEach(element => {
+        if (typeof element == "number") {
+            numArray.push(element)
+        }
+    });
+    let sum = numArray.reduce(function (a, b) {
```

Comment:
Also nicely done 
you can make it fast using one loop also to do both checking if it's number and adding it